src/ApiPort.VisualStudio.Common/Analyze/AnalysisOptions.cs

```diff
@@ -28,7 +30,7 @@ public AnalysisOptions(string description, IEnumerable<string> inputAssemblies,
 
         public string Description { get; }
 
-        public IEnumerable<IAssemblyFile> InputAssemblies { get; }
+        public ImmutableDictionary<IAssemblyFile, bool> InputAssemblies { get; }
```

Comment:
Why are you changing this? Can't you create a field here `skipAssembliesIfPackagesAreFound` and reference that where needed? The only reason why it would make sense to change this (IMHO) would be if each assembly could have a different value for the bool

Reply:
The bool is per assembly.